### ISSUE TITLE
Image is in _ipx folder and can't be found

### ISSUE THREAD

**Issue by JosefHjorth** [nuxt/image#354]
When building my solution it tries to reference the images in a missing _ipx folder.

```
<source type="image/webp" 
srcset="/_ipx/images/stock_collage.png?w=320&amp;f=webp 320w, 
/_ipx/images/stock_collage.png?w=640&amp;f=webp 640w, 
/_ipx/images/stock_collage.png?w=768&amp;f=webp 768w, 
/_ipx/images/stock_collage.png?w=700&amp;f=webp 700w, 
/_ipx/images/stock_collage.png?w=700&amp;f=webp 700w" 
sizes="(max-width: 320px) 100vw, (max-width: 640px) 100vw, (max-width: 768px) 100vw, (max-width: 1024px) 700px, 700px">
```

This is my (relevant) configuration. I've tried with the default configuration, but whatever I do nuxt-image tries to reference _ipx folder.

```
export default {
  ssr: false,
  target: "static",
  buildModules: [
    '@nuxt/image',
  ],
  image: {
    staticFilename: '[publicPath]/images/[name]-[hash][ext]'
  },
}
```

This is the html element:
`
<nuxt-picture format="webp" sizes="xl:700 lg:700 md:100vw sm:100vw xs:100vw" src="/images/stock_collage.png" />
`

Why is nuxt-image referencing non existent _ipx folder?

**Comment by JosefHjorth** [comment]
I noticed there was a similar issue reported here; https://github.com/nuxt/image/issues/215 , but I'm running 0.5.0 already

**Comment by shadow81627** [comment]
Is the issue with `ssr: false,`? I believe the `_ipx` route is a server side endpoint that does the image manipulation. Without the server running the static image provider (ipx) can't be accesses to generate the static images?

**Comment by JosefHjorth** [comment]
Thanks! Might be, but does that mean I can't use nuxt/image since I use client side rendering only?

**Comment by shadow81627** [comment]
I believe the `@nuxt/image` static provider (ipx) requires [full static setup](https://nuxtjs.org/blog/going-full-static#new-config-option-target).

You may need to consider another `@nuxt/image` provider such as [cloudinary](https://cloudinary.com/) to host the images and do image transformations.

**Comment by JosefHjorth** [comment]
Switched to cloudinary and it worked much better. Thanks.

**Comment by ChristopherCapito** [comment]
@shadow81627 cant the IPX part be run during the generation step? like, open the server for IPX; generate the images, close it down, and then we are left with the images plus a client-only / full static generated site?

**Comment by shadow81627** [comment]
Yes, I run ipx with static target. You might need to have SSR on for the ipx server to run during generate.

**Comment by salv0** [comment]
So it seems that having 
```
target: 'static'
ssr: false
```

breaks nuxt-image with the default provider.
This also happens in 0.5.0.

Any solution besides using another provider as suggested above?
I'd just like to run the ipx optimizations on generate command and have all ready in the dist folder to be moved to production :)

**Comment by elglogins** [comment]
Any updates here?

**Comment by WieFel** [comment]
+1

**Comment by dinkar030501** [comment]
any update?

**Comment by oxedom** [comment]
?

**Comment by hbsoftco** [comment]
+1

**Comment by clever-honey** [comment]
Any update? I'm running a Nuxt3 App with ssr false.

**Comment by parallized** [comment]
Any update? I'm running a Nuxt3 App with ssr false.

**Comment by dmitrevnik** [comment]
Any update? I'm running a Nuxt3 App with ssr false.
